indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +68,39 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHEditTrack(RHManageTracksBase):
+
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.track = Track.find_one(id=request.view_args['id'])
```

Comment:
:warning: Security issue - you can put any track ID from another event here. Set an URL normalization rule on the RH to prevent this!

```python
    normalize_url_spec = {
        'locators': {
            lambda self: self.track
        }
}
```

That way any request to a track not matching its locator data (which should contain track id and event id) will automatically redirect to the proper location where the access check is performed for the correct event.